### PR TITLE
Add port Sonic 3 AIR

### PR DESCRIPTION
**Pull request by JeodC**
Submitting on behalf of spec58.

# New Port for Sonic 3 AIR

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/sonic3air/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='utf-8'?>
+<gameList>
+  <game>
+    <path>./Sonic 3 AIR.sh</path>
+    <name>Sonic 3 AIR</name>
+    <desc>A fan-made widescreen remaster of Sonic 3 & Knuckles.</desc>
```
Comment (kloptops) [inline ports/sonic3air/gameinfo.xml:6]:
```suggestion
    <desc>A fan-made widescreen remaster of Sonic 3 &amp; Knuckles.</desc>
```

File: ports/sonic3air/sonic3air/config/logfile.txt
Comment (kloptops) [inline ports/sonic3air/sonic3air/config/logfile.txt:1]: Log file shouldn't be included?

File: ports/sonic3air/sonic3air/log.txt
Comment (kloptops) [inline ports/sonic3air/sonic3air/log.txt:1]: same with this.

**Review by kloptops — CHANGES_REQUESTED** [review]
Lots of assets are included but no license to say they're okay to distribute?

A few log files can be deleted.

**Review by kloptops — APPROVED** [review]
👍

**Review by JanTrueno — APPROVED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**